documentscanner/src/main/res/color-night/chip_text.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
-    <item android:color="#121212" android:state_checked="true" />
+    <item android:color="@color/dark_grey" android:state_checked="true" />
     <item android:color="#888A89" />
```

Comment:
If this should use a defined colour reference instead?

Reply:
I think so. @Gnzlt please could you take a look?